src/components/HTTPHeader.jsx

```diff
@@ -64,8 +64,20 @@ export default function HTTPHeader({ type, path }) {
       setTimeout(() => setCopy(false), 3000);
     }
   }, [copied]);
-  const BASE_URL = "https://top.gg/api/";
-  const fullUrl = new URL(path, BASE_URL).href;
+  const BASE_URL = "https://top.gg/api";
+  if (path.startsWith(BASE_URL)) {
+    path = path.replace(BASE_URL, "");
+  }
+  if (path.startsWith("api")) {
+    path = path.replace("api", "");
+  }
+  else if (path.startsWith("/api")) {
+    path = path.replace("/api", "")
+  }
+  else if (!path.startsWith("/")) {
+    path = `/${path}`
+  }
+  const fullUrl = new URL(`api${path}`, BASE_URL).href;
```

Comment:
Have you tested this? the `api` prefix still prevents us from passing fully qualified domain names into the component like
```html
<HTTPHeader path="https://google.com" />
```
probably not something we need for a while but like I said it could prevent more refactoring in the future